### ISSUE TITLE
Address "All of the organizations will also be disabled for the time being." from mockup

### ISSUE THREAD

**Issue by jeremyf** [forem/forem#16893]
In reviewing the mockup, there's a surprise statement:

"All of the organizations will also be disabled for the time being."  We need to flesh out what that means.

<img width="903" alt="Screen Shot 2022-03-16 at 9 49 19 AM" src="https://user-images.githubusercontent.com/2130/158604939-709e4669-593f-41b7-a9b9-ffcd9a361947.png">

**Comment by amywtlin** [comment]
Ah. My bad, I think I forgot to mention this. 

That means that once comment-only is on, people can't create new posts on behalf of an organization anymore. Given that org members highly likely will not be admins, I think it is alright that we don't handle this. The only scenario that can be weird is to when an admin who also belongs to an org to post as on behalf of the org. But I think we can leave this to be handled in as a post-launch fix.

cc @anujbhavsar96 to double-check on the behavior of organizations when comment-only is on, as it's been a while since we last chatted about it.

**Comment by jeremyf** [comment]
Rephrase "Only admins will be able to post on behalf of organizations."

**Comment by jeremyf** [comment]
I have amended a pull request to modify the language.  I am closing this issue as we have clarity on what the text means.

**Comment by jeremyf** [comment]
Posted to provide an additional example: https://forem-team.slack.com/archives/C02C32DLPJP/p1648064143298039

> @amy As I’m wrestling with the dashboard, it occurred to me that we may have a disconnect in what we mean by “Only admins can post”.
> 
> Here’s the scenario, User A is part of Org 1.  User A is not a site-wide admin nor super admin.  However, User A is an admin of Org 1.  By existing logic for AuthN/AuthZ, User A cannot create posts (neither a post as an individual nor as part of an organization).
> 
> I want to confirm that as acceptable for Use Case 1-1.  Based on our past conversation, I feel comfortable that this is acceptable; however I want to raise this example from a different perspective.
> 
> tl;dr - The number of “admin” concepts is too high

**Comment by amywtlin** [comment]
> By existing logic for AuthN/AuthZ, User A cannot create posts (neither a post as an individual nor as part of an organization).

Correct. Only site-wide admins and super admins are allowed to post when a Forem is comment-only. Org admins cannot post anymore, unless they also happen to be a super admin or a side-wide admin. If this is the case, there is a hole we decide not to fill where these super admins/side-wide admins can post on behalf of an org. This is the only scenario where an org admin can post in a comment-only Forem, as mentioned [here](https://github.com/forem/forem/issues/16893#issuecomment-1071942358).

>tl;dr - The number of “admin” concepts is too high

I agree 100% with this and I intend to use more specific terms to refer to them going forward. I have something defined [here](https://www.notion.so/forem/Glossary-product-communication-e1561558d5194ba78fdbfe6b3ba810b3). (for example, I call "site-wide admins" "cross-site admins" in there - but they are not finalized and I actually like site-wide admins more.) Feel free to comment on suggestions and we can finalize the terms and start using specific, consistent terms.
